binaries/conda/build_packages.py

```diff
@@ -1,23 +1,44 @@
-import os
-import sys
 import argparse
-import subprocess
+import os
+from datetime import date
 
 conda_build_dir = os.path.dirname(os.path.abspath(__file__))
 REPO_ROOT = os.path.join(conda_build_dir, "..", "..")
-MINICONDA_DOWNLOAD_URL = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
-CONDA_BINARY = os.popen("which conda").read().strip() if os.system(f"conda --version") == 0 else  f"$HOME/miniconda/condabin/conda"
+MINICONDA_DOWNLOAD_URL = (
+    "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
+)
+CONDA_BINARY = (
+    os.popen("which conda").read().strip()
+    if os.system(f"conda --version") == 0
+    else f"$HOME/miniconda/condabin/conda"
+)
 
 if os.name == "nt":
-    #Assumes miniconda is installed in windows
+    # Assumes miniconda is installed in windows
     CONDA_BINARY = "conda"
 
+
+def add_nightly_suffix_conda(ts_version, ma_version, wa_version):
```

Comment:
Let's refactor this function into

```python
def add_nightly_suffix_conda(binary_name : str) -> str:
   todays_date = date.today().strftime("%Y%m%d")
   return binary_name + ".dev" + todays_date
```

Later down we can use `[add_nightly_suffix(binary_name) for binary_name in [ts, ma, wa]]`

Reply:
Done